marathon-template

```diff
@@ -24,10 +24,15 @@ def all_apps_on(marathon_uri):
   app_ids = map(lambda app: app["id"].strip("/"), apps)
   return app_ids
 
+def is_healthy_task(task):
+  result = len(filter(lambda healthCheck: not healthCheck["alive"], task["healthCheckResults"])) == 0
+  return bool(result)
```

Comment:
Do we need this? If not, the comparison can be inlined into return statement itself.